resources/assets/coffee/react/_components/beatmapset-panel.coffee

```diff
@@ -21,6 +21,11 @@
 el = React.createElement
 
 class @BeatmapsetPanel extends React.Component
+  mixins: [React.addons.PureRenderMixin]
```

Comment:
this doesn't do anything because
- you need to use `React.createClass`
- you overrode the only thing it does anyway (`shouldComponentUpdate`)

:expressionless: